datastore_orm/model.py

```diff
@@ -687,12 +694,15 @@ def get(self, key, missing=None, deferred=None,
 
         :raises: :class:`ValueError` if eventual is True and in a transaction.
         """
+        start = datetime.datetime.now()
         entities = self.get_multi(keys=[key],
                                   missing=missing,
                                   deferred=deferred,
                                   transaction=transaction,
                                   eventual=eventual, model_type=model_type)
         if entities:
+            end = datetime.datetime.now()
+            print('Time taken for get {}'.format(end - start))
             return entities[0]
 
     def get_multi(self, keys, missing=None, deferred=None,
```

Comment:
have you tried using the get_multi method from the base class?

Reply:
Yes I had tried, it is quite fast too but it worked faster with threading approach, probably because get_multi is called each time with a single key.